sections/memory_layout.tex

```diff
@@ -69,15 +85,15 @@ \section{Block Packed Storage}\label{sec: memory layout}
 	\begin{algorithm}
 		\caption{Calculation of \(\lambda_n\) and Simulation of \(Z_{n+1}\)}	
 		\label{algo: calc lambda and simulate Z_{n+1}}
-		\(\xS \leftarrow \sigma_n\)\tcp*[f]{red cache}\;
+		\(\xS \leftarrow \sigma_n^2\)\tcp*[f]{red cache}\;
 		\(\xZ \leftarrow 0_{k\times 1}\)\tcp*[f]{blue cache}\;
 		\For{\(i = 0,\dots,n-1\)}{
-			\(\xl \leftarrow L_{\frac{n(n+1)}2+i}\)\tcp*[f]{teal cache}\;
-			\(\xS = \xS - \xl\xl^T\)\;
+			\(\xl \leftarrow L_{n,i}\)\tcp*[f]{teal cache}\;
+			\(\xS \leftarrow \xS - \xl\xl^T\)\;
 			\(\xY \leftarrow Y_i\)\tcp*[f]{violet cache}\;
-			\(\xZ = \xZ + \xl^T \xY\)\;
+			\(\xZ \leftarrow \xZ + \xl^T \xY\)\;
```

Comment:
der mix von = und \leftarrow war tatsächlich absichtlich. Arrows für memory movement in den cache und heraus, = für operationen auf dem cache.

Reply:
(weiß nicht ob das eine gute Idee war)

Reply:
Kann mit einer anderen Operation als \leftarrow leben, aber bei = ist meine Assoziation immer ein Gleichungssystem, also "finde `Z` für `Z = Z +l^T Y`"...

Reply:
okay - ich hänge nicht an den =. Dachte dass sie es vielleicht übersichtlicher machen, aber wenn das nicht der fall ist lassen wir das :)